repo/repo.go

```diff
@@ -25,6 +25,8 @@ type Repo interface {
 	// SetAPIAddr sets the API address in the repo.
 	SetAPIAddr(addr string) error
 
+	Self() Repo
```

Comment:
Ah, I see now. I incorrectly assumed the fsrepo.Open constructor returned an fsrepo